core/domain/exp_services.py

```diff
@@ -361,6 +463,9 @@ def get_exploration_ids_matching_query(query_string, cursor=None):
 def get_non_private_exploration_summaries():
     """Returns a dict with all non-private exploration summary domain objects,
     keyed by their id.
+
+    Returns:
+        A python dict with all non-private ExplorationSummary domain objects.
```

Comment:
Omit "python" -- it's kind of obvious that we're in python-land.

What is the format of this dict? I can't tell from what's written. See above for how to describe a dict -- I gave an example in one of the previous review comments.